src/js/materia/materia.coms.json.js

```diff
@@ -111,6 +111,11 @@ Namespace('Materia.Coms').Json = (() => {
 		return _sendRequest('POST', url, JSON.stringify(dataObject))
 	}
 
+	// newer XMLHttpRequest json api
+	const del = (url, dataObject) => {
+		_sendRequest('DELETE', url, dataObject)
+	}
```

Comment:
`dataObject` needs to be assigned a default value (`{}`, as in the declaration of the `post` method above) and turned into a JSON string prior to being passed into `_sendRequest`.